### PR TITLE
Hw 6 classes and context manager

### PR DESCRIPTION
**Pull request by multiset9**

### CODE REVIEW

File: hw-6_Classes_and_Context_manager.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# У вас есть список(list) IP адрессов. Вам необходимо создать
+# класс который будет иметь методы:
+
+
+class IpProcessing:
+    def __init__(self, ip_addresses):
+        self.ip_addresses = ip_addresses
+
+    def get_reversed_ip(self):
+        reversed_ip = []
```
Comment (mmykhailenko) [inline hw-6_Classes_and_Context_manager.py]: better use list comprehension

File: hw-6_Classes_and_Context_manager.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        for ip in self.ip_addresses:
+            reversed_ip.append(f'{ip} -> {".".join(ip.split(".")[::-1])}')
+        return reversed_ip
+
+    def get_ip_without_first_octet(self):
+        without_first_octet = []
```
Comment (mmykhailenko) [inline hw-6_Classes_and_Context_manager.py]: better use list comprehension

File: hw-6_Classes_and_Context_manager.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        for ip in self.ip_addresses:
+            without_first_octet.append(f'{ip} -> {".".join(ip.split(".")[1::])}')
+        return without_first_octet
+
+    def get_last_octet(self):
+        last_octet = []
```
Comment (mmykhailenko) [inline hw-6_Classes_and_Context_manager.py]: better use list comprehension

**Review by mmykhailenko — APPROVED** [review]

File: hw-6_Classes_and_Context_manager.py (hunk 1 above)
Comment (multiset9) [reply hw-6_Classes_and_Context_manager.py]: Thanks, fixed

File: hw-6_Classes_and_Context_manager.py (hunk 3 above)
Comment (multiset9) [reply hw-6_Classes_and_Context_manager.py]: Thanks, fixed

File: hw-6_Classes_and_Context_manager.py (hunk 2 above)
Comment (multiset9) [reply hw-6_Classes_and_Context_manager.py]: Thanks, fixed

**Pull request merged**